src/keycloak/keycloak_admin.py

```diff
@@ -762,6 +762,46 @@ def update_user(self, user_id, payload):
         )
         return raise_error_from_response(data_raw, KeycloakPutError, expected_codes=[204])
 
+    def disable_user(self, user_id):
+        """Disable the user from the realm. Disabled users can not log in.
+
+        :param user_id: User id
+        :type user_id: str
+
+        :return: Http response
+        :rtype: bytes
+        """
+        return self.update_user(user_id=user_id, payload={"enabled": False})
+        
+    def enable_user(self, user_id):
+        """Enable the user from the realm.
+
+        :param user_id: User id
+        :type user_id: str
+
+        :return: Http response
+        :rtype: bytes
+        """
+        return self.update_user(user_id=user_id, payload={"enabled": True})
+    
+    def disable_all_users(self):
+        """Disable all existing users.
+        """
+        users = self.get_users()
+        for user in users:
+            user_id = user["id"]
+            print(f"Disabling user with id: {user_id}")
```

Comment:
No `print`s please :) I do not want to force unstoppable logs to the users of the library. You can use `logging` with a combination of a `verbose` parameter or just a `verbose` parameter itself keeping the prints, but the default should be 0 verbosity :)

Reply:
Good point. I removed the print statements.